openfisca_france/param/param.xml

```diff
@@ -2781,7 +2784,16 @@
         <VALUE deb="2010-01-01" fin="2011-12-31" valeur="878" />
         <VALUE deb="2012-01-01" fin="2012-12-31" valeur="960" />
         <VALUE deb="2013-01-01" fin="2013-12-31" valeur="1016" />
-        <VALUE deb="2014-01-01" fin="2014-12-31" valeur="1021" />
```

Comment:
Well except if I'm totally off track the new decote threshold has two parameters in 2014 one for singles and one for couples. That parameters might have been entered before the implementation of the reform (i'll check with gitk... )